examples/remarketing/upload_call_conversion.py

```diff
@@ -0,0 +1,174 @@
+                "Partial error ocurred: "
+                f"'{upload_call_conversions_response.partial_failure_error.message}'"
+            )
+
+        # Print the result if valid.
+        uploaded_call_conversion = upload_call_conversions_response.results[0]
+        if uploaded_call_conversion.call_start_date_time:
+            print(
+                "Uploaded call conversion that occurred at "
+                f"'{uploaded_call_conversion.call_start_date_time}' "
+                f"for caller ID '{uploaded_call_conversion.caller_id}' "
+                "to the conversion action with resource name "
+                f"'{uploaded_call_conversion.conversion_action}'."
+            )
+    except GoogleAdsException as ex:
+        print(
+            f"Request with ID '{ex.request_id}'' failed with status "
+            f"'{ex.error.code().name}' and includes the following errors:"
+        )
+        for error in ex.failure.errors:
+            print(f"\tError with message '{error.message}'.")
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+if __name__ == "__main__":
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description="Imports offline call conversion values for calls related "
+        "to your ads."
+    )
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument(
+        "-c",
+        "--customer_id",
+        type=str,
+        required=True,
+        help="The Google Ads customer ID.",
+    )
+    parser.add_argument(
+        "-a",
+        "--conversion_action_id",
+        type=str,
+        required=True,
+        help="The ID of the conversion action to upload to.",
+    )
+    parser.add_argument(
+        "-i",
+        "--caller_id",
+        type=str,
+        required=True,
+        help="The caller ID from which this call was placed. Caller ID is "
+        "expected to be in E.164 format with preceding '+' sign, "
+        "e.g. '+16502531234'.",
+    )
+    parser.add_argument(
+        "-s",
+        "--call_start_date_time",
+        type=str,
+        required=True,
+        help="The date and time at which the call occurred. The format is "
+        "'yyyy-mm-dd hh:mm:ss+|-hh:mm', e.g. '2019-01-01 12:32:45-08:00'.",
+    )
+    parser.add_argument(
+        "-t",
+        "--conversion_date_time",
+        type=str,
+        required=True,
+        help="The the date and time of the conversion (should be after the "
+        "click time). The format is 'yyyy-mm-dd hh:mm:ss+|-hh:mm', e.g. "
+        "'2019-01-01 12:32:45-08:00'.",
+    )
+    parser.add_argument(
+        "-v",
+        "--conversion_value",
+        type=str,
```

Comment:
not sure if a string is most appropriate here. The value should be a currency value (https://support.google.com/google-ads/answer/3419241?hl=en) so let the parser filter out bad data like characters.

Reply:
Fixed.